intercom2/python-lib/intercomapi.py

```diff
@@ -0,0 +1,83 @@
+import json
+import urlparse
+import requests
+from requests.packages.urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+
+# Basic logging
+def log(*args):
+    for thing in args:
+        if type(thing) is dict:
+            thing = json.dumps(thing)
+        print('Intercom plugin - %s' % thing)
+
```

Comment:
Why not using the logging library?
That way, the elements will be properly colored in the logs output :)

```
import logging
logging.basicConfig(level=logging.INFO, format='%(asctime)s %(levelname)s %(message)s')
```

Reply:
Sure! Indeed that would be better.
Are you ok if I replace with that?

```
import logging
# Basic logging
logging.basicConfig(level=logging.INFO, format='%(asctime)s %(levelname)s %(message)s')
def log(*args):
    for thing in args:
        if type(thing) is dict:
            thing = json.dumps(thing)
        logging.info('Intercom plugin - %s' % thing)
```

Reply:
I think you could just replace your log() by logging.info() as string formatting already displays dicts properly 😄 
(and set "Intercom plugin" in the logging basicConfig format, see below)

``` python 
>>> import logging
>>> logging.basicConfig(level=logging.INFO, format='%(asctime)s %(levelname)s Intercom plugin - %(message)s')
>>> dict = {'foo': 'bar', 'pickle': 'rick'}
>>> logging.info('test %s' % dict)
"2018-04-05 10:15:53,348 INFO Intercom plugin - test {'foo': 'bar', 'pickle': 'rick'}"
```